File: graffiti/src/main/java/com/ue/graffiti/model/PenShapeItem.java
package com.ue.graffiti.model;

import com.ue.adapterdelegate.Item;

/**
 * Created by hawk on 2018/1/17.
 */

public class PenShapeItem implements Item {
    public int flag;
    public int image;
    //多type RecyclerView中有作用
    public int index;
    public String name;
    public boolean isChecked;

    public PenShapeItem(int flag, int image, String name) {
        this.flag = flag;
        this.image = image;
        this.name = name;
    }
}

File: graffiti/src/main/java/com/ue/graffiti/model/PenShapeItem.kt
package com.ue.graffiti.model

/**
 * Created by hawk on 2018/1/17.
 */

class PenShapeItem(var flag: Int, var image: Int, var name: String) {
    //多type RecyclerView中有作用
    var index = 0
    var isChecked: Boolean = false
}
